mysql/etc/containerbuddy.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
It looks like this isn't referenced anywhere. Perhaps it's best to let https://github.com/tgross/triton-mysql be the authoritative repo for the MySQL image?